sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -21,6 +20,7 @@ class KeyClient(_KeyVaultClientBase):
             :end-before: [END create_key_client]
             :language: python
             :caption: Creates a new instance of the Key client
+            :dedent: 4
```

Comment:
we are adding back the dedents and now they are different across the file?

Reply:
Yep, I rendered the docs and saw class examples were one level too deep, method examples two levels too deep.